### PR TITLE
Remove JAVA implementation and fix for working with Ballerina latest version

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
- Remove all the JAVA implementations of the package
- Update the file and folder structure
- Fix for working with Ballerina latest version
- Add documentations

### CODE REVIEW

File: soap/tests/test.bal [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+endpoint Client soapClient {
+    clientConfig: {
+        url: "http://localhost:9000"
+    }
+};
+
+@test:Config
+function testSendReceive() {
+    io:println("\n ---------------------------------------------------------------------------");
```
Comment (shafreenAnfar) [inline soap/tests/test.bal]: Do we really need this line ?

Comment (ldclakmal) [reply soap/tests/test.bal]: No. Will remove this.

File: README.md [hunk 2]
```diff
@@ -1,80 +1,50 @@
-# Ballerina Soap Connector
+# SOAP Connector
 
-The Soap connector allows you to send a ordinary xml request to a soap backend by specifying the necessary details to construct a soap envelope. It abstracts out the details of the creation of a soap envelope, headers and the body in a soap message.
+The SOAP connector allows you to send a ordinary XML request to a soap backend by specifying the necessary details to
```
Comment (Bhashinee) [inline README.md]: send an ordinary XML request

File: soap/soap_utils.bal [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.package soap;
+
+import ballerina/io;
+import ballerina/http;
+import ballerina/time;
+import ballerina/mime;
```
Comment (Bhashinee) [inline soap/soap_utils.bal]: Shall we arrange these imports in alphabetical order?

**Pull request merged**